### DISCUSSION TITLE
Works like a charm with QCells (Gen4) Inverter & Batterie

### DISCUSSION THREAD

**Discussion by Funkyflosh** [wills106/homeassistant-solax-modbus#117] [Show and tell]
@wills106, thank you so much fot this great work! It works like a dream from the start! 👍

**Comment by wills106** [comment]
What's the exact Model?

Is it a Q.VOLT HYB-G3-3P which is a rebranded SolaX Gen4. Does your Serial start with H34 or is it a different one?
I just want to understand which QCells work and then I can update the documentation and also the Branding so people can find this Integration better.

**Reply by Funkyflosh** [reply]
That's correct, a Q.VOLT HYB-G3-3P and SN# starts with H34. 
I'd be glad to give something back to this project. If there's any testing needed on this unit, please don't hesitate to ask!

**Reply by wills106** [reply]
As it's just a rebranded SolaX Gen4 there isn't much testing left to do.
Just glad you have fed back it works on the Q.VOLT HYB-G3-3P
